drainer/config.go

```diff
@@ -75,9 +75,62 @@ type SyncerConfig struct {
 	DoDBs             []string           `toml:"replicate-do-db" json:"replicate-do-db"`
 	DestDBType        string             `toml:"db-type" json:"db-type"`
 	Relay             RelayConfig        `toml:"relay" json:"relay"`
-	EnableDispatch    bool               `toml:"enable-dispatch" json:"enable-dispatch"`
-	SafeMode          bool               `toml:"safe-mode" json:"safe-mode"`
-	EnableCausality   bool               `toml:"enable-detect" json:"enable-detect"`
+	// disable* is keep for backward compatibility.
+	// if both setted, the disable one take affect.
+	DisableDispatchFlag *bool `toml:"disable-dispatch-flag" json:"disable-dispatch-flag"`
+	EnableDispatchFlag  *bool `toml:"enable-dispatch-flag" json:"enable-dispatch-flag"`
+	DisableDispatchFile *bool `toml:"disable-dispatch" json:"disable-dispatch"`
```

Comment:
how about use these configs in integration test

Reply:
add in https://github.com/pingcap/tidb-binlog/pull/915/commits/505451b4944d87c9d42b833e939458de0903d40f